playbooks/tests/tasks/main.yml

```diff
@@ -1,17 +1,37 @@
 ---
 - include: all.yml
+
 - include: controller.yml
+
 - include: network.yml
```

Comment:
Seems like some tests in network.yml will also fail if not run on controller hosting l3 active router